TableViewKit/Protocols/Selectable.swift

```diff
@@ -18,7 +18,7 @@ extension Selectable {
     public func select(in manager: TableViewManager, animated: Bool, scrollPosition: UITableViewScrollPosition = .none) {
 
```

Comment:
I'm not sure, I don't know if that will grow and we would add many other methods to the TableViewManager. Anyways we can take that account 👍